packages/components/organisms/f-cookie-banner/CHANGELOG.md

```diff
@@ -19,6 +22,14 @@ v0.24.0
 - Moved `isBodyHeightLessThanWindowHeight` to computed so it can re-calculate cookie position.
 
 
+Latest (to be added to next release)
```

Comment:
Nit: This section looks a bit out of place

Reply:
Is it ok to make the CHANGELOG updates as part of a follow-up PR.

Will just be quicker than waiting for the build and approving all the visual regression changes again!